src/webapp/vos_data.py

```diff
@@ -151,11 +150,16 @@ def _expand_oasis_legacy_managers(managers):
                 new_managers[name]["DNs"] = {"DN": data["DNs"]}
             else:
                 new_managers[name]["DNs"] = None
-        return {"Manager": expand_attr_list(new_managers, "Name", ordering=["ContactID", "Name", "DNs"],
+
+            new_managers[name]["CILogonID"] = None
+            if self.contacts_data:
+                user = self.contacts_data.users_by_id.get(data["ID"])
+                if user:
+                    new_managers[name]["CILogonID"] = user.cilogon_id
+        return {"Manager": expand_attr_list(new_managers, "Name", ordering=["Name", "CILogonID", "DNs"],
```

Comment:
Just to confirm - does this mean that now with `CILogonID` we stop caring about `ContactID` ?

Reply:
I deleted ContactID because it never existed. I don't know how it got there -- maybe I meant "ID"? But ID is for internal use and we don't expose it in the rgsummary and vosummary XMLs, only miscuser.